js/google_ima.js

```diff
@@ -425,7 +425,8 @@ require("../html5-common/js/utils/utils.js");
           //displaying it, so just notify AMC that we are showing an overlay.
           else
           {
-            _amc.notifyNonlinearAdStarted(this.currentAMCAdPod.id);
+            // raise WILL_PLAY_NONLINEAR_AD event and notify nonlinear ad started
+            _amc.sendURLToLoadAndPlayNonLinearAd(this.currentAMCAdPod, this.currentAMCAdPod.id, this.currentAMCAdPod.ad.tag_url);
```

Comment:
Also, do we no longer need to call notifyNonlinearAdStarted?  Isn't that what is triggering the WILL_PLAY_NONLINEAR_AD event?  And actually, if we're calling that, do we need to call sendURLToLoadAndPlayNonLinearAd at all or is sendURLToLoadAndPlayNonLinearAd replacing notifyNonlinearAdStarted?